### DISCUSSION TITLE
RSS feeds in the new forced-to-"blog view" paradigm

### DISCUSSION THREAD

**Discussion by arcanetrivia** [AprilSylph/XKit-Rewritten#888] [Lounge]
This is a bit of a vent; sorry for that, I just figure people who are interested enough in technical stuff that they are following a repo on Github might either be feeling the same pain, or have suggestions.

I subscribe to all Tumblr blogs I follow by RSS, and do my actual reading of them that way, not on the dashboard. A couple of days ago I tried to follow a new-to-me blog that is one of the ones that's been forced into "blog view" because they must not have that "custom theme" toggle turned on. And I discovered a new problem with this change: **you can't get at the RSS feed for such a blog anymore.** https://[blogname.]tumblr.com/rss just forwards to https://www.tumblr.com/[blogname], which my RSS reader rightfully complains is not valid XML. (Blogs with custom themes still work as they always did; the RSS is right there.)

I submitted a support ticket asking if this was an intentional removal of functionality or just an oversight, which they first misunderstood as "your friends will have to submit their own support tickets" and then told me unhelpfully "well it's up to the blog owner to turn on custom theme". I could not get a straight answer about whether Tumblr development did this intentionally or at least knew it would happen and considered it acceptable, rather than something that needed to be handled. The answer to "how do I get at such a blog's RSS feed?" seems to be "You can't." Which splits the site into blogs that can be followed that way and blogs that can't, which, to put it briefly, sucks.

Now, affected blogs that I previously subscribed to this way haven't had their RSS feeds cut off. So it seems to me like there still must be a way -- i.e. the RSS generator hasn't been totally removed, if only I knew how to get at it. Do any of you big brains have any ideas how it might be done?

**Comment by AprilSylph** [comment]
What blog are you trying to subscribe to? Blogs that don't have a custom theme should still have RSS feeds at `https://*.tumblr.com/rss`, just like how their archives still exist at `https://*.tumblr.com/archive`. For example, even though `best-of-reblogs` doesn't have a custom theme, I can access https://best-of-reblogs.tumblr.com/rss (and https://best-of-reblogs.tumblr.com/archive) just fine.

If a blog has checked "Hide [blogname] from people without an account", though, that's a different story. That's a privacy setting, and there are no exemptions for that, even if it means breaking certain features like the RSS feed and the archive. This isn't new.

**Comment by arcanetrivia** [comment]
I noticed that about the archives (when I saw people panicking about that), which is why I was so confused/annoyed about RSS... But it seems the blog in question that made me notice (https://www.tumblr.com/hillsarehollow) _is_ in fact set to "You'll need to be on Tumblr to see this particular blog." I guess it's just chance that I never before tried to subscribe to such a blog by RSS ever in 11 years 😯 and it just happened to semi-coincide with the move to the non-subdomain style.

eta: You know what, actually, I can think of one time I did try, some years ago, because I had a friend from DeviantArt who made a Tumblr and hid it in this way. I tried to subscribe to her feed and as I recall, it appeared to work from the RSS reader side, but the feed was just blank and never contained any posts, rather than complaining that https://[whatever].tumblr.com/rss wasn't XML. I guess possibly the fowarding behavior has introduced a new wrinkle to that. 

Well, anyway. I've been told, now 😉 thanks.
